cypress/support/api/circulation.js

```diff
@@ -20,3 +20,25 @@ Cypress.Commands.add('createItemCheckinApi', (data) => {
     body: data,
   });
 });
+
+Cypress.Commands.add('editCirculationRulesApi', (data) => {
+  cy
+    .okapiRequest({
+      method: 'PUT',
+      path: 'circulation/rules',
+      body: data,
+    });
+});
+
+Cypress.Commands.add('getCirculationRulesApi', () => {
```

Comment:
move api methods to related fragments

Reply:
Do you mean in some fragment we need to add some method which uses `getCirculationRulesApi` command? Could you please explain why we need it?

Reply:
let's imagine there is a fragment with list and button to add some object into the table. It logically to have gui and api methods related with adding of such objects into the system. In smoke tests we interact with folio fragments. And if we need to do something with folio object, we should start to work with method of related fragment. I hope this comment will be useful.